### PR TITLE
Lesson 7 Task 2 - Capital letters

### PR DESCRIPTION
**Pull request by jean-farinea**
1. Class ConvertToCapital created with a method to convert text to capital.

2. Test class created to verify if method is working as expected, including null and empty sources.

### CODE REVIEW

File: src/main/java/lesson7/task2/ConvertToCapital.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package lesson7.task2;
+
+public class ConvertToCapital {
```
Comment (strzeleckik) [inline src/main/java/lesson7/task2/ConvertToCapital.java]: for example TextConverter sounds better ;)

File: src/main/java/lesson7/task2/ConvertToCapital.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+    public static String convertTextToCapital(String text) {
+        String convertedText = text.toUpperCase();
```
Comment (strzeleckik) [inline src/main/java/lesson7/task2/ConvertToCapital.java]: will it work with null? in my opinion it should throw NULL POINTER EXCEPTION

**Review by strzeleckik — CHANGES_REQUESTED** [review]
to check

**Comment by jean-farinea** [comment]
Changes done, please review.

File: src/test/java/lesson7/task2/TextConverterTest.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TextConverterTest {
+
+    @ParameterizedTest
+    @NullAndEmptySource
+    @ValueSource(strings = {"first text", "second text", "third text"})
+    void shouldCheckIfTextIsConvertedToCapital(String text) {
```
Comment (strzeleckik) [inline src/test/java/lesson7/task2/TextConverterTest.java:16]: positive and negative scenario check in one test. should be splitted into separate methods or refactored

**Comment by jean-farinea** [comment]
Test adjusted, please review.

**Review by strzeleckik — APPROVED** [review]
ok

**Pull request merged**